Cargo.lock

```diff
@@ -1965,11 +1964,11 @@ dependencies = [
 [[package]]
 name = "parity-crypto"
 version = "0.1.0"
-source = "git+https://github.com/paritytech/parity-common#0045887fecd2fec39e56c962a0b27e2caf3b9474"
+source = "git+https://github.com/paritytech/parity-common#51258799ee639783a7153d807ce9ddf4d49b29fa"
 dependencies = [
  "ethereum-types 0.3.2 (registry+https://github.com/rust-lang/crates.io-index)",
  "quick-error 1.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
- "ring 0.12.1 (git+https://github.com/paritytech/ring)",
+ "ring 0.12.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
what's the reason to change `ring` dependency? iirc the official repo does not contain several changes that we need

Reply:
`ring` is problematic since it does not allow to link multiple versions of itself into program, and at Golem we are using `ring 0.13`

 https://github.com/briansmith/ring/blob/7e3682c02170f5eafa423b63bacfa018de2878a9/Cargo.toml#L14-L15

Reply:
my final workaround is to extremely strip parity-ethereum project under [my private branch](https://github.com/tworec/parity-ethereum/tree/strip-down-ethkeys) to just contain `ethkey` and `ethstore` which we are using as for now.

Is there any other solution to have linked ring 12 and 13 in single program?

Reply:
So for this PR I'm reverting this change